_health-care/_js/components/form-elements/ErrorableCheckbox.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import _ from 'lodash';
+
+/**
+ * A form checkbox with a label that can display error messages.
+ *
+ * Validation has the following props.
+ * `errorMessage` - Error string to display in the component.
+ *                  When defined, indicates checkbox has a validation error.
+ * `label` - String for the checkbox label.
+ * `required` - boolean. Render marker indicating field is required.
+ * `value` - Boolean. Whether or not the checkbox is checked.
+ * `onValueChange` - a function with this prototype: (newValue)
+ */
+class ErrorableCheckbox extends React.Component {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.inputId = _.uniqueId('errorable-checkbox-');
+  }
+
+  handleChange(domEvent) {
+    this.props.onValueChange(domEvent.target.checked);
+  }
+
+  render() {
+    // TODO: extract error logic into a utility function
+    // Calculate error state.
+    let errorSpan = '';
+    let errorSpanId = undefined;
+    if (this.props.errorMessage) {
+      errorSpanId = `${this.inputId}-error-message`;
+      errorSpan = <span className="usa-input-error-message" id={`${errorSpanId}`}>{this.props.errorMessage}</span>;
+    }
+
+    // Calculate required.
+    let requiredSpan = '';
+    if (this.props.required) {
+      requiredSpan = <span className="usa-additional_text">Required</span>;
+    }
+
+    return (
+      <div className={`usa-input-grid usa-input-grid-large ${this.props.errorMessage ? 'usa-input-error' : ''}`}>
+        <input
+            aria-describedby={errorSpanId}
+            checked={this.props.value === true}
```

Comment:
Drop the `=== true`.

`this.props.value` is by contract (which is enforced by propTypes in development mode) is only ever a boolean.